### ISSUE TITLE
Add progressViewOffset prop

### ISSUE THREAD

**Issue by AlanSalas** [hyochan/react-native-masonry-list#47]
**Is your feature request related to a problem? Please describe.**
Hi devs, we use this library in our project and we want to pass the prop progressViewOffset to the mansory list component but you dont declare it in the props and as you know that prop allow us to move the progress indicator from the scrollview.

**Describe the solution you'd like**
Just add the prop progressViewOffset

I write this like an issue because i tried to fork and add the prop myself but when add the fork to the package.json of my project it throws me an error, so i hope you read this and consider this. Thank you.

**Comment by hyochan** [comment]
Thanks for the issue! Fixed in `1.4.0`.
You may use `refreshControlProps` to handle that.

**Comment by hotaryuzaki** [comment]
> Thanks for the issue! Fixed in `1.4.0`. You may use `refreshControlProps` to handle that.

how to use that? i cont see in docs
like this?
```
   ...
   refreshControlProps={
          <RefreshControl
            refreshing={refreshing}
            onRefresh={_onRefresh}
            progressViewOffset={300}
          />
   }
   ...
```
